for_003_nrt_rw1_glad_deforestation_alerts/start.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
+
+#Change the NAME variable with the name of your script
+NAME=for_003_nrt_rw1
```

Comment:
It is helpful to have a descriptive word here so that when we get errors, i know this is GLAD alerts without having to check what dataset for.003 is
```suggestion
NAME=for_003_nrt_rw1_glad
```